scripts/zones/Korroloka_Tunnel/IDs.lua

```diff
@@ -25,7 +25,11 @@ zones[xi.zone.KORROLOKA_TUNNEL] =
         ENTERED_SPRING                = 7339,  -- The water in this spring is pleasant and tepid. This looks like a nice place to warm yourself up.
         LEFT_SPRING_EARLY             = 7340,  -- You are not warm enough yet. You will need to spend more time than that in the spring to get your body heated up.
         LEFT_SPRING_CLEAN             = 7341,  -- Your whole body is piping hot, and the smell of the Rafflesia pollen is gone!
+        FILL_FLASK                    = 7342,  -- You carefully draw the water stored in the clam into <keyitem>
```

Comment:
This may seem nit-picky, but the text here should be `You carefully draw the water stored in the clam into %.` (note the '.') - where % is a wildcard, so you can put `<keyitem>` or whatever into there.

We use this tooling: https://github.com/LandSandBoat/UpdateExtractor/tree/main
during the client update process and if these comment strings don't match _exactly_ they won't get updated. At best this shows up in a git diff and we fix it manually, at worst it gets missed completely and then we have "client ID shift" baked into these IDs until someone spots it.

Reply:
Don't pay attention to the actual number, these IDs are a bit out of date, but the text is still relevant: https://github.com/LandSandBoat/UpdateExtractor/blob/main/out/scripts/zones/Korroloka_Tunnel/Text.lua.raw.txt#L7257